src/applications/personalization/profile360/components/PersonalInformation.jsx

```diff
@@ -12,11 +12,21 @@ function Gender({ gender }) {
   let content = 'This information is not available right now.';
   if (gender === 'M') content = 'Male';
   else if (gender === 'F') content = 'Female';
-  return <span>{content}</span>;
+  return (
+    <div data-field-name="gender">
+      <h3>Gender</h3>
+      <div>{content}</div>
+    </div>
+  );
 }
 
 function BirthDate({ birthDate }) {
-  return <span>{moment(birthDate).format('MMM D, YYYY')}</span>;
+  return (
+    <div data-field-name="birth-date">
+      <h3>Birth date</h3>
+      <div>{moment(birthDate).format('MMM D, YYYY')}</div>
+    </div>
+  );
 }
 
```

Comment:
I just went ahead and cleaned this up by moving the label into the component. The whole component should probably be moved into its own file anyway.